app/views/layouts/_edit_form_buttons.html.haml

```diff
@@ -10,6 +10,7 @@
 - restful              ||= false
 - show_cancel_button = %w(user_edit user_update menu_tree zone_edit zone_update ldap_seq_edit rbac_group_edit rbac_group_update rbac_group_field_changed category_edit category_update label_tag_mapping_edit label_tag_mapping_update timeprofile_edit timeprofile_update timeprofile_copy)
 - force_cancel_button ||= false
+- return if action_url == "tagging_edit"
```

Comment:
Umm, having a `return` in a view is evil and I am afraid it can cause side effects in the HAML parsing.